src/gui/PasswordGeneratorWidget.cpp

```diff
@@ -54,6 +62,12 @@ PasswordGeneratorWidget::PasswordGeneratorWidget(QWidget* parent)
         m_ui->entropyLabel->setFont(defaultFont);
         m_ui->strengthLabel->setFont(defaultFont);
     }
+
+    m_ui->comboBoxWordSeparator->addItems(QStringList() <<" " <<"#" <<";" <<"-" <<":" <<"." <<"@");
```

Comment:
@TheZ3ro shouldn't there be spaces around the `<<` operators?
`<< " " << "#" << ";" << "-" << ":" << "." << "@"`